### ISSUE TITLE
Some trajs being missed in new soln

### ISSUE THREAD

**Issue by markmac99** [markmac99/WesternMeteorPyLib#44]
Some trajectories seem to be getting missed
Further examination shows that its the ignore-station code. If too many stations are marked ignore, then the solver fails immediately. I need to add more stations to be sure it'll function properly. 
Experimentation shows that it runs *faster* with 15 stations than with 6, because it spends less time retrying already-excluded stations. Further work needed on this.

**Comment by markmac99** [comment]
Some issues were due to a bug introduced by the efforts to improve performance - the code to exclude cameras was causing the solver to entirely fail in edge cases. I've fixed this and it seems to be catching more cases. 
However, the code behaves slightly differently on my PC and Hydrid2. Both have Python 3.8, but on hydrid2, the code fails to solve a trajectory that is successfully solved on my PC. I'd like to test it on centaurid as well.
